xml/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerProvider.xml

```diff
@@ -30,7 +30,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>The provider for the <see cref="T:Microsoft.Extensions.Logging.EventSource.EventSourceLogger" />.</summary>
```

Comment:
Maybe keep the description here but remove the link?
EventSourceLogger is internal: https://github.com/aspnet/Logging/blob/master/src/Microsoft.Extensions.Logging.EventSource/EventSourceLogger.cs

Reply:
I originally did that, but then realized that the summary is not adding anything that the name of the type doesn't already tell you.

Reply:
I'd agree with @gewarren here.

Reply:
You probably don't have compliance requirements here so I guess it's fine.